io/io/inc/ROOT/RRawFile.hxx

```diff
@@ -50,6 +50,8 @@ public:
       ELineBreaks fLineBreak = ELineBreaks::kAuto;
       /// Read at least fBlockSize bytes at a time. A value of zero turns off I/O buffering. A negative value indicates
       /// that the protocol-dependent default block size should be used.
+      /// After construction, a negative block size is used to store the block size value when caching is turned off
+      /// (see `SetBuffering()`).
```

Comment:
The alternating between buffering and caching can be confusing.  Are they the same in this context?

Reply:
Yes, I'll reword.